airflow/providers/google/cloud/operators/life_sciences.py

```diff
@@ -90,5 +92,10 @@ def execute(self, context: 'Context') -> dict:
             api_version=self.api_version,
             impersonation_chain=self.impersonation_chain,
         )
-
-        return hook.run_pipeline(body=self.body, location=self.location, project_id=self.project_id)
+        result = hook.run_pipeline(body=self.body, location=self.location, project_id=self.project_id)
+        LifesciencesLink.persist(
+            context=context,
+            task_instance=self,
+            project_id=self.project_id or hook.project_id,
+        )
```

Comment:
do we have ID of the pipeline in the `result`? if yes we maybe we have possibility to create link directly to it not to all the pipelines?

Reply:
Yes, we have ID,
but we can´t link to the details of a single pipeline (we can only see the list of all pipelines)

Reply:
Btw, we dont need result variable I think - as we dont use it at all, only for return purpose. You can just persist before the return statement